Source/Scene/Scene.js

```diff
@@ -539,6 +541,18 @@ define([
          */
         this.fog = new Fog();
 
+        this._sunCamera = new Camera(this);
```

Comment:
Is this member really needed?  It is only used in one place after the constructor.